src/patternfly/layouts/Flex/examples/flex-alignment-example.hbs

```diff
@@ -1,109 +1,198 @@
-  {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
+  {{#> l-flex-item l-flex-item--modifier="pf-m-align-right"}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
 {{/l-flex}}
 
 {{#> example-title}}Align right on multiple groups.{{/example-title}}
-{{#> l-flex l-flex--modifier="example-border"}}
+{{#> l-flex}}
   {{#> l-flex newcontext}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
   {{/l-flex}}
   {{#> l-flex newcontext l-flex--modifier="pf-m-align-right"}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
   {{/l-flex}}
   {{#> l-flex newcontext l-flex--modifier="pf-m-align-right"}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
   {{/l-flex}}
 {{/l-flex}}
 
 {{#> example-title}}Using <code>.pf-m-flex-1</code> to align adjacent content.{{/example-title}}
 {{#> l-flex}}
-  {{#> l-flex newcontext l-flex--modifier="pf-m-flex-1 example-border"}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
-    {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
+  {{#> l-flex newcontext l-flex--modifier="pf-m-flex-1"}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
   {{/l-flex}}
-  {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
-  {{#> example-text example-text--modifier="example-border"}}Flex item{{/example-text}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
 {{/l-flex}}
 
 {{#> example-title}}Using <code>.pf-m-flex-1</code> to align adjacent content.{{/example-title}}
```

Comment:
Should we give this a different description from the example above to call out the difference? Or is this example necessary?